docs/gui/release_notes_de.md

```diff
@@ -4,6 +4,9 @@ Im Folgenden finden Sie die Release Notes für das User Management.
 
 Technische Details zu einem Release finden Sie im zugehörigen [Changelog](https://docs.cloudogu.com/de/docs/dogus/usermgt/CHANGELOG/).
 
+## Release 1.15.2-2
+* Verbesserung von Fehlernachrichten sowohl in manuellen als auch CSV-Import-Prozessen.
```

Comment:
```suggestion
* Verbesserung von Fehlernachrichten sowohl in manuellen als auch CSV-Import-Prozessen.
```
It would be nice to clarify like in english what a manual process refers to, in english you use "manual editing". This nuance disappeared from the german translation.

Reply:
- [x] done